crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/AbstractLockCommand.java

```diff
@@ -86,6 +90,7 @@ public static AbstractLockCommand getLockCommand(String request) {
 
     @Override
     public CallStatus run() {
+        RequestMetrics.Context context = metrics.startLockRequest(getLockCommandName(), domain);
```

Comment:
Does this have to be done at the HTTP layer?

I understand if locking is special, just wondering.